rancher-config/docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: '2'
+
+services:
+    image: nhsuk/pa11y-dashboard:${DOCKER_IMAGE_TAG}
+    database:
+        image: mongo:3.3
+    dashboard:
+        build:
+            context: .
+        ports:
+            - 3333:3333
+            - 3000:3000
+        environment:
+            WEBSERVICE_DATABASE: mongodb:${MONGODB_URL}
+        links:
+            - database
+    labels:
+      traefik.enable: true
+      traefik.frontend.rule: $TRAEFIK_RULE
+      traefik.backend: ${RANCHER_STACK_NAME}-pa11y-dashboard
+      traefik.port: 3000
+      prometheus.port: 3000
```

Comment:
There isn't going to be any metrics for prometheus to scrape in this image.

Reply:
Removed `prometeus`